scormxblock/static/html/scormxblock.html

```diff
@@ -10,14 +11,19 @@
     player_config_attrs += 'data-{{}}="{{}}" '.format(item[0], cgi.escape(item[1], quote=True))
 %>
 
+<%
+should_show_description = ('{self.description}' != '') or ({show_popup_manually})
```

Comment:
I think this condition will be removed in next story related to background grey area. So, we can keep it for now to meet the AC of this ticket.